src/utils/getHydratedTemplateString.ts

```diff
@@ -0,0 +1,33 @@
+import { CSSGen } from '../services/css-gen/css-gen';
+import generateStylesJS from './generateStylesJS';
```

Comment:
@jyash97 everything looks good, one final change I would suggest is that to follow the file naming scheme of `generate-styles-js.js` rather than `generateStyleJS.js` as it can cause issues in case insensitive filesystems.

Reply:
This file existed already and the naming scheme follows the same pattern in the `utils` folder

Reply:
ohh ok got it, then we will handle that in a separate issue and PR